include/dpp/automod.h

```diff
@@ -83,10 +83,6 @@ enum automod_trigger_type : uint8_t {
 	 * @brief Keyword filtering
 	 */
 	amod_type_keyword = 1,
-	/**
-	 * @brief Harmful/malware links
-	 */
-	amod_type_harmful_link = 2,
```

Comment:
removed by discord

Reply:
fair play, is it completely removed and not marked deprecated?

Reply:
discord removed it completely from their docs. i've re added it here and marked it as deprecated.